### PR TITLE
Proto for events and rejections change

### PR DESCRIPTION
**Pull request by YuriiHaidamaka**
In this PR we added order date in the MenuNotAvailable message and corrected all documentation in the next files : 
- `events.proto`
- `rejections.proto`

### CODE REVIEW

File: model/src/main/proto/javaclasses/mealorder/c/events.proto [hunk 1]
```diff
@@ -41,33 +41,33 @@ import "javaclasses/mealorder/values.proto";
...
     spine.time.LocalTime po_daily_deadline = 6;
 }
 
-// An event fired upon the vendor description update.
+// An event fired upon a vendor description update.
 //
 message VendorUpdated {
 
-    // An identifier of the updated vendor.
+    // An identifier of a updated vendor.
```
Comment (vladislav-kozachenko) [inline model/src/main/proto/javaclasses/mealorder/c/events.proto]: // An identifier of an updated vendor.

File: model/src/main/proto/javaclasses/mealorder/c/rejections.proto [hunk 2]
```diff
@@ -47,113 +48,116 @@ message VendorAlreadyExists {
...
 
-    // A Date when a command was rejected.
+    // A time point when it was rejected.
     google.protobuf.Timestamp when_rejected = 4;
 }
 
 // The rejection to create order.
 //
-// It is thrown if the order already exists.
+// It is thrown if an order already exists.
```
Comment (vladislav-kozachenko) [inline model/src/main/proto/javaclasses/mealorder/c/rejections.proto]: Maybe we need to note there that it was attempt to create order with the same user, date and vendor as already exists.

Comment (YuriiHaidamaka) [reply model/src/main/proto/javaclasses/mealorder/c/rejections.proto]: Is it better?
"Order for this user on the same date and for the same vendor already exists."

File: model/src/main/proto/javaclasses/mealorder/c/events.proto [hunk 3]
```diff
@@ -41,33 +41,33 @@ import "javaclasses/mealorder/values.proto";
...
 
-    // An identifier of the user, who added vendor.
+    // An identifier of a user, who added a vendor.
     UserId who_added = 2;
 
     // Time of addition.
     google.protobuf.Timestamp when_added = 3;
 
-    // An email adress  to send an purchase order.
+    // An email adress to send an purchase order.
```
Comment (YegorUdovchenko) [inline model/src/main/proto/javaclasses/mealorder/c/events.proto]: ...to send **a** purchase order

File: model/src/main/proto/javaclasses/mealorder/c/events.proto [hunk 4]
```diff
@@ -41,33 +41,33 @@ import "javaclasses/mealorder/values.proto";
...
     spine.net.EmailAddress email = 4;
 
-    // A phone numbers of the vendor.
+    // Phone numbers of a vendor.
     repeated PhoneNumber phone_numbers = 5;
 
-    // The time to which all orders to this vendor must be made.
+    // Time to which all orders to this vendor must be made.
```
Comment (YegorUdovchenko) [inline model/src/main/proto/javaclasses/mealorder/c/events.proto]: to - > till?

File: model/src/main/proto/javaclasses/mealorder/c/events.proto [hunk 5]
```diff
@@ -255,13 +255,13 @@ message PurchaseOrderValidationOverruled {
...
-    // A user identifiers that overruled the validation.
+    // An identifier of a user that overruled a validation.
     UserId who_overruled = 2;
 
-    // The reason why validation is overruled.
+    // A reason why validation is overruled.
     string reason = 3;
 
-    // Time when validation was overruled.
+    // A point of time when validation was overruled.
```
Comment (YegorUdovchenko) [inline model/src/main/proto/javaclasses/mealorder/c/events.proto]: point -> moment?

File: model/src/main/proto/javaclasses/mealorder/c/events.proto [hunk 6]
```diff
@@ -227,10 +227,10 @@ message PurchaseOrderSent {
...
     google.protobuf.Timestamp when_sent = 2;
 
-    // An E-mail address of the sender.
+    // An E-mail address of a sender.
     spine.net.EmailAddress sender_email = 3;
 
-    // An E-mail address of the recipient.
+    // An E-mail address of a recipient.
     spine.net.EmailAddress vendor_email = 4;
 }
```
Comment (YegorUdovchenko) [inline model/src/main/proto/javaclasses/mealorder/c/events.proto]: Maybe we should add more detailed explication? 
Purchase order validation may complete with failure if some orders contain large amount of the same dish. That may indicate user mistake.  Such orders are attached to this event.

File: model/src/main/proto/javaclasses/mealorder/c/events.proto [hunk 7]
```diff
@@ -255,13 +255,13 @@ message PurchaseOrderValidationOverruled {
     // A purchase order identifier.
     PurchaseOrderId id = 1;
 
-    // A user identifiers that overruled the validation.
+    // An identifier of a user that overruled a validation.
```
Comment (YegorUdovchenko) [inline model/src/main/proto/javaclasses/mealorder/c/events.proto]: that -> who?

File: model/src/main/proto/javaclasses/mealorder/c/rejections.proto [hunk 8]
```diff
@@ -47,113 +48,116 @@ message VendorAlreadyExists {
     // A name of existing vendor.
     VendorName vendor_name = 2;
 
-    // A Date when a command was rejected.
+    // A time point when it was rejected.
```
Comment (YegorUdovchenko) [inline model/src/main/proto/javaclasses/mealorder/c/rejections.proto]: Earlier it was "A point of time"

**Pull request merged**